cypress/objects/loginPage.js

```diff
@@ -0,0 +1,26 @@
+class LoginPage {   //klasiu pav is didziosios
+    visit() {
+        cy.visit('/');  //homepage
+    }
+    getValidationError() {
+        return cy.get('.Select.not-valid');
+    }
+    getSubmit() {
+        return cy.get('[type="submit"]');
+    }
+    getUserID() {
+        return cy.get('[id="loginForm.userId"]');
+    }
+    getUserOption(userName) {
+        return cy.get(`[aria-label="${userName}"]`);
+    }
+    getRole() {
+        return cy.get('[id="loginForm.role"]');
+    }
+    getRoleOption() {
```

Comment:
Here you could do exactly what you did with `getUserOption` and insert a parameter like `roleName`, so that it can be reused not only with the 'User' role, but with others as well :)